docs/pipelines/how-it-works/index.md

```diff
@@ -1,76 +1,89 @@
+### Invoker Lambda
+
+The [Invoker Lambda](https://github.com/gruntwork-io/terraform-aws-ci/blob/main/modules/ecs-deploy-runner/main_lambda.tf)
+is a lambda function written in python that acts as the AWS entrypoint for your pipeline.
+It has 3 primary roles:
+
+1. It serves as a gatekeeper for pipelines runs, determining if a particular command is allowed to be run
+2. It creates ECS tasks that run terraform, docker, or packer commands
+3. It ships deployment logs back to your CI/CD tool
+
+### Standard Configuration
 
-And the people did praise Kaniko, for it did not require root privileges, and could be run by users without fear of compromising the system. It did not store secrets in the Docker daemon, but instead used a secure file system to store them, ensuring that they could not be accessed by unauthorized parties.
+The ECS deploy runner is flexible and can be configured for many tasks.  The [standard configuration](https://github.com/gruntwork-io/terraform-aws-ci/tree/main/modules/ecs-deploy-runner-standard-configuration)
+is a set of 4 ECS task definitions that we ship with Pipelines by default.
+Once you have your pipeline deployed you can [modify](../maintain/extending.md) the configuration as you like.
+The configuration defines what scripts are accepted by the invoker lambda and which arguments may be provided. The invoker lambda
+will reject ANY script or argument not defined in the ECS Deploy Runner configuration.
+The 4 default tasks are defined below.
 
-### Terraform Planner/Applier
+#### Docker Image Builder (Kaniko)
 
-In the realm of deployment, there exist powerful tools that rival the strength and cunning of even the most feared warriors. Among them, the ECS Deploy Runner, Terraform Planner, and Applier stand tall, ready to unleash their might upon the field of battle.
+The Docker Image Builder task definition allows CI jobs to build docker images.
+This ECS task uses an open source library called [Kaniko](https://github.com/GoogleContainerTools/kaniko) to enable docker builds from within a docker container.
+We provide a [docker image](https://github.com/gruntwork-io/terraform-aws-ci/tree/main/modules/ecs-deploy-runner/docker/kaniko) based on kaniko for this task.
 
-The ECS Deploy Runner is like a fierce direwolf, swift and agile, capable of executing complex deployment tasks with ease. It is the tool of choice for those who seek to deploy their applications with speed and precision.
+#### Packer AMI Builder
 
-The Terraform Planner is like a wise and ancient dragon, possessing an almost infinite knowledge of the infrastructure it seeks to conquer. With its keen intellect, it can survey the land and plan the deployment with utmost accuracy, leaving no stone unturned in its quest for victory.
+The Packer AMI Builder task definition allows CI jobs to build AMIs using Hashicorp's Packer tool. This task runs in
```

Comment:
```suggestion
The Packer AMI Builder task definition allows CI jobs to build AMIs using HashiCorp's Packer tool. This task runs in
```

Reply:
Super nit

Reply:
My hobby is triggering the people who create branding guidelines 😉